atomics/T1539/T1539.yaml

```diff
@@ -77,3 +77,38 @@ atomic_tests:
       remove-item #{output_file}
     name: powershell
     elevation_required: false
+- name: Steal Chrome Cookies via Remote Debugging (Mac)
+  auto_generated_guid: 
+  description: |-
+    The remote debugging functionality in Chrome can be used by malware for post-exploitation activities to obtain cookies without requiring keychain access. By initiating Chrome with a remote debug port, an attacker can sidestep encryption and employ Chrome's own mechanisms to access cookies.
+
+    If successful, this test will output a list of cookies.
+
+    Note: Chrome processes will be killed during this test.
+  
+    See https://posts.specterops.io/hands-in-the-cookie-jar-dumping-cookies-with-chromiums-remote-debugger-port-34c4f468844e
+  supported_platforms:
+  - macos 
+  dependency_executor_name: bash
+  dependencies:
+  - description: 'go and whitechocolatemacadamianut '
+    prereq_command: |-
+      go version
+      /tmp/WhiteChocolateMacademiaNut/chocolate -h
+    get_prereq_command: |-
+      brew install go --force
```

Comment:
Can you also split the requirements into two dependencies ? One to install `go` and another one to clone `WhiteChocolateMacademiaNut` ?

Reply:
Split the two out into separate dependencies.